issue_template.md

```diff
@@ -0,0 +1,18 @@
+Please include the following information before submitting your issue
+Any information not included may be requested when diagnosing the issue
+
+Description of the Issue
+------------------------
+
+
+Version (Google Play, Branch Name)
```

Comment:
Maybe add something like "git hash when built from source"?

Reply:
@baka0815 That's unnecessary. If they are using Play, that has a definitive hash. If they are listing a branch, the hash is always "newest" unless they specify otherwise. There is no need to make the template an essay. It is VERY general tags for what info is relevant.